spark-jobs/src/main/scala/filodb/repair/PartitionKeysCopier.scala

```diff
@@ -75,25 +74,32 @@ class PartitionKeysCopier(conf: SparkConf) {
     Option(schemas.part.binSchema.partitionHash(partKey.partKey, UnsafeUtils.arayOffset))
 
   val numOfShards: Int = sourceDatasetConfig.getInt("num-shards")
-  private val isDownsampleRepair = conf.getBoolean("spark.filodb.partitionkeys.copier.isDownsampleCopy", false)
-  private val repairStartTime = parseDateTime(conf.get("spark.filodb.partitionkeys.copier.repairStartTime"))
-  private val repairEndTime = parseDateTime(conf.get("spark.filodb.partitionkeys.copier.repairEndTime"))
-  private val diskTimeToLiveSeconds = if (isDownsampleRepair) {
-    val dsSettings = new DownsamplerSettings(rawSourceConfig)
-    val highestDSResolution = dsSettings.rawDatasetIngestionConfig.downsampleConfig.resolutions.last
+  private val isDownsampleCopy = conf.getBoolean("spark.filodb.partitionkeys.copier.is.downsample.copy", false)
+  private val copyStartTime = parseDateTime(conf.get("spark.filodb.partitionkeys.copier.start.time"))
+  private val copyEndTime = parseDateTime(conf.get("spark.filodb.partitionkeys.copier.end.time"))
```

Comment:
I would recommend having "ingestionTime" in property name so operations understand we are repairing by ingestion time which is different for raw and downsample datasets.

Reply:
I'll add this in a separate pr.